thirdparty/clusterapi/api/v1alpha4/cluster_phase_types.go

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
I would really love to see some info in the commit message that provides context on why we have chosen to keep this in thirdparty directory. 
also we should have some doc on how to keep this in sync with upstream.